test/tests/functional/pbs_node_rampdown.py

```diff
@@ -6945,6 +6818,9 @@ def test_release_multi_jobs(self):
                             {'resources_available.ncpus': 3},
                             id=self.hostC)
 
+        #Wait for nodes to come up
+        time.sleep(10)
+        self.license_count_match(3)
```

Comment:
You've changed every license_count_match() to 3.  I don't know much about the new licensing code, but is it changing at all?  Is it 3 before the release?  If so, shouldn't we just remove the check from all the tests?

Reply:
@bhroam License count remains same unless we delete any node from cluster. You are right, I will remove this check from code.